File: algoexpert/classPhotos.py
def classPhotos(redShirtHeights, blueShirtHeights):
    # Write your code here.
	sortedRed=sorted(redShirtHeights)
	sortedBlue=sorted(blueShirtHeights)
	bluefront=True
	redfront=True
	for i in range(len(sortedRed)):
		if sortedRed[i]<=sortedBlue[i]:
			redfront=False
		if sortedBlue[i]<=sortedRed[i]:
			bluefront=False
		
	if bluefront or redfront:
		return True
	
    return False

File: InterviewTraining/js/classPhotos.js
function classPhotos(redShirtHeights, blueShirtHeights) {
    const sortedRed = redShirtHeights.slice().sort((a, b) => a - b);
    const sortedBlue = blueShirtHeights.slice().sort((a, b) => a - b);
    let blueFront = true;
    let redFront = true;

    for (let i = 0; i < sortedRed.length; i++) {
        if (sortedRed[i] <= sortedBlue[i]) {
            redFront = false;
        }
        if (sortedBlue[i] <= sortedRed[i]) {
            blueFront = false;
        }
    }

    return blueFront || redFront;
}

module.exports = classPhotos;
